solr/core/src/java/org/apache/solr/schema/FieldProperties.java

```diff
@@ -85,7 +93,7 @@ static int propertyNameToInt(String name, boolean failOnError) {
         return 1 << i;
       }
     }
-    if (failOnError && !"default".equals(name)) {
+    if (failOnError && !IGNORED_PROPERTY_NAMES.contains(name)) {
```

Comment:
can be this a method with switch case?

Reply:
Indeed a method "isPropertyIgnored" is clearer, thanks.
However I prefer a simple triple String equals rather than a switch, because I find the first more concise and actually faster for only 3 strings.